packages/typespec-ts/test/modularUnit/apiOperations.spec.ts

```diff
@@ -624,7 +624,6 @@ describe("api operations in Modular", () => {
         
         export function createTesting(
           endpointParam: string,
-          apiVersion: string,
```

Comment:
why we have this change?

Reply:
https://github.com/Azure/typespec-azure/pull/1597/files

Reply:
Confirmed with Chenjie, this is introduced by tcgc change. @v-jiaodi could you revert this change and skip this test case for now?

issue tracked: https://github.com/Azure/autorest.typescript/issues/2867